crates/sui-core/src/transaction_orchestrator.rs

```diff
@@ -632,7 +632,7 @@ impl TransactionOrchestratorMetrics {
             )
             .unwrap(),
             wait_for_finality_finished: register_int_counter_with_registry!(
-                "tx_orchestrator_wait_for_finality_fnished",
+                "tx_orchestrator_wait_for_finality_finished",
```

Comment:
This will likely impact the dashboard so we'd need to make sure it gets fixed there too or just keep it as is.

Reply:
seconded - we have a lot of typos that we have intentionally not fixed because fixing all the dashboards is such a pain

Reply:
Let's keep as is!